### PR TITLE
Counting_sort in Cpp, C,  Js

### PR DESCRIPTION
**Pull request by SHARC-99**
Here is the code for Counting sort in C, Cpp, Python, and JavaScript along with a readme file with a brief explanation about counting sort.

### CODE REVIEW

**Comment by ankitapuri** [comment]
Can you please delete the python file we already have that

**Comment by SHARC-99** [comment]
> Can you please delete the python file we already have that

ok.. but please tell me that do I need to re-push the PR...becoz I am new to github

**Comment by ankitapuri** [comment]
yes

**Comment by SHARC-99** [comment]
> yes

done!

**Comment by SHARC-99** [comment]
Does it have any problem?
please tell me if there are some!
or do I need to do something extra!

File: Counting_Sort/counting_sort.c [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+} 
+
+// Driver program to test above function 
+int main() 
+{ 
+	char arr[] = "HACKTOBERFEST"; //"applepp"; 
+
+	countSort(arr); 
+
+	printf("Sorted character array is %sn", arr);
```
Comment (ankitapuri) [inline Counting_Sort/counting_sort.c]: Here an extra n gets printed out

File: Counting_Sort/counting_sort.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    }
+    for (i = min; i <= max; i++) {
+        while (count[i] > 0) {
+            arr[j] = i;
+            j++;
+            count[i]--;
+        }
+    }
+    return arr;
+};
```
Comment (ankitapuri) [inline Counting_Sort/counting_sort.js:20]: Add a driver code for this and add in the numeric form

Comment (ankitapuri) [reply Counting_Sort/counting_sort.js:20]: Add driver code in js file . I mentioned previously as well.

**Comment by ankitapuri** [comment]
Add driver code in js file . I mentioned previously as well.

**Comment by SHARC-99** [comment]
Now I think it's done! I resolved all the problems. Sorry for the inconvenience.

**Review by ankitapuri — APPROVED** [review]

**Pull request merged**